routes/api/stories.js

```diff
@@ -0,0 +1,43 @@
+//2
+const express = require('express');
+
+
+const {asyncHandler} = require('../utils');
+const { PostLike } = require("../../db/models");
+const { restoreUser } = require('../../auth');
+const createError = require('http-errors');
+
+
+
+const router = express.Router();
+
+//Like a story
+router.post('/stories/:id/likes', restoreUser, asyncHandler(async(req, res, next) => {
+    if (!res.locals.authenticated) {
+       return next(createError(401)); 
+    }
+    const { userId, postId } = req.body;
+
+    const likedPost = await PostLike.create({
+        userId: res.locals.user.id,
+        postId: req.params.id
+    })
+
+    //res.send(req.body);
+
+    res.json({message: 'Success'});
+
+
+}));//works
+
+//Unlike a story
+router.delete('/stories/likes/:id', restoreUser, asyncHandler(async(req, res, next) => {
+    await PostLike.destroy({
```

Comment:
Check to make sure the user is authenticated and that the user is the owner of that like (`res.locals.user.id === like.userId`)

Here is a starter:
```js
if (!res.locals.authenticated) {
   return next(createError(401)); 
}
```